ee/desktop/runner/runner.go

```diff
@@ -721,3 +732,33 @@ func iconFilename() string {
 func (r *DesktopUsersProcessesRunner) iconFileLocation() string {
 	return filepath.Join(r.usersFilesRoot, iconFilename())
 }
+
+func (r *DesktopUsersProcessesRunner) refreshOsqueryData() {
+	osqueryVersion := "Unknown"
```

Comment:
Right now, when desktop first starts, the menu item will display as `osquery Version: `. Upon the query result being returned it will change to `osquery Version: <the version number>` or `osquery Version: Unknown` in case of an error.

Is `osquery Version: ` what we want for that waiting-for-osquery state? It could be something like `osquery Version: Waiting for osquery...` 🤷

Reply:
I don'y think it matters much. `Unknown` or `Waiting...`